bigquery/google/cloud/bigquery/table.py

```diff
@@ -204,6 +204,20 @@ def table_id(self):
         """
         return self._table_id
 
+    @property
+    def reference(self):
```

Comment:
I recently added this property to `TableListItem` as well. I'll update this PR to share this property with that implementation before submitting.